jenkins-build-driver/src/main/java/org/jboss/pnc/jenkinsbuilddriver/BuildJob.java

```diff
@@ -35,19 +44,19 @@ public boolean configure(RunningEnvironment runningEnvironment, boolean override
         try {
             job = jenkinsServer.getJob(jobName);
         } catch (IOException e) {
-            throw new BuildDriverException("Cannot check for existing job.", e);
+            log.info("No job exists, so there will be created new one.");
```

Comment:
I would expect IOException when something went wrong not when the job does not exists. This exception should be re-thrown.